lago/cmd.py

```diff
@@ -398,6 +400,45 @@ def do_console(prefix, host, **kwargs):
 
 
 @lago.plugins.cli.cli_plugin(
+    help='Create Ansible configuration of the environment'
+)
+@in_lago_prefix
+@with_logging
+def do_generate_ansible_hosts(prefix, out_format, **kwargs):
+    """
+    This method iterate through all the VMs and create an ansible
+    hosts in current directory. It defines an IP address
+    and private key for the machine in group named by its type,
+    for example:
+
+    [ovirt-host]
+    192.168.201.2 ansible_host=192.168.201.2 ansible_ssh_private_key_file=/path/id_rsa
+    192.168.201.4 ansible_host=192.168.201.4 ansible_ssh_private_key_file=/path/id_rsa
+    [ovirt-engine]
+    192.168.201.3 ansible_host=192.168.201.3 ansible_ssh_private_key_file=/path/id_rsa
+    """
+    inventory = defaultdict(list)
+    for vm in prefix.virt_env.get_vms().values():
+        vm_mgmt_ip = [
+            nic.get('ip')
+            for nic in vm.nics() if 'management' in nic.get('net')
+        ]
+        for ip in vm_mgmt_ip:
+            inventory[vm._spec['vm-type']].append(
+                "{ip} ansible_host={ip} ansible_ssh_private_key_file={ssh_key}\n".format(
+                    ip=ip,
+                    ssh_key=prefix.virt_env.prefix.paths.ssh_id_rsa()
+                )
+            )
+
+    with open('./ansible_hosts', 'w+') as f:
```

Comment:
I think either allowing to pass a file descriptor as parameter or printing to stdout would be nice, allowing you to do something like:
```
lago ansible_hosts > my_file
```